components/Schedule.js

```diff
@@ -0,0 +1,170 @@
+import styled from "styled-components";
+import { useRouter } from "next/router";
+import { useEffect, useState } from "react";
+import Link from "next/link";
+
+const StyledDiv = styled.div`
+  text-align: center;
+`;
+const StyledImg = styled.img`
+  border-radius: 5px;
+  width: 100px;
+  height: 100px;
```

Comment:
you could use `aspect-ratio: 1 / 1;` instead setting the height of the image. Not necessary but could be nice.